x500/base/src/test/java/org/wildfly/security/x500/X500PrincipalBuilderTest.java

```diff
@@ -18,7 +18,7 @@
 
 package org.wildfly.security.x500;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEqua;
```

Comment:
I think this is supposed to be `assertEquals`?

Reply:
Yeah, sorry just making tiny changes to figure out merge issues